src/outline.jl

```diff
@@ -173,6 +218,12 @@ function str_value(x)
         x.kind == CSTParser.Tokens.AT_SIGN && return "@"
         x.kind == CSTParser.Tokens.DOT && return "."
         return ""
+    elseif x.kind === CSTParser.Tokens.TRIPLE_STRING
+        return string("\"\"\"", x.val, "\"\"\"")
+    elseif x.kind === CSTParser.Tokens.STRING
+        return string("\"", x.val, "\"")
+    elseif x.kind === CSTParser.Tokens.CHAR
+        return string("\'", x.val, "\'")
```

Comment:
I believe you don't need this check for `CSTParser.Tokens.CHAR` or just return `x.val`, since `x.val` already contains `'` as is.

Reply:
Indeed. I only checked the two string cases and assumed chars are handled the same...